### ISSUE TITLE
User interaction with Jupyterhub

### ISSUE THREAD

**Issue by Shreyanand** [operate-first/odh-moc-support#26]
I'm creating this issue to understand and perhaps refine user interaction with JH on Operate first.

Currently, this is the workflow for me if I'm the user:

1. I read a blog on the website and it takes me to an instance of Jupyter hub
2. I login using my google account
3. I select the image corresponding to the blog I read
4. A container is spawned and I see the notebook directory in my PVC

A few days later, I read another blog on operate first and it takes me to another  notebook image.

1. I login using my google account
2. I'm no longer shown a spawn screen because the container is already running for my account
3. I stop my server, start it again and select the new image
4. Now, I see two notebook directories in my PVC. 
5. I select the directory with the name "notebook-latest-date"

Do we want to refine this workflow?
Some ideas:
- Instead of naming the directory "notebook-timestamp" we could call them "image_name-timestamp"
- Do we shut down containers and free up PVC as the user exits? Do we want the user interaction to be temporary or do we want to keep a user profile?

**Comment by SankBad** [comment]
Hi Shrey, 

I created the same [issue](https://github.com/operate-first/odh-moc-support/issues/24) in the morning. 
Later found out this [issue](https://github.com/AICoE/aicoe-ci/issues/82) already exists in `AICoE/aicoe-ci` repo

**Comment by SankBad** [comment]
The above comment is regarding the 1st idea.

**Comment by 4n4nd** [comment]
@Shreyanand We need to rebuild all our notebook images so that the existing images in the registry are updated with new ones that have this fix.

**Comment by 4n4nd** [comment]
> Do we shut down containers and free up PVC as the user exits? Do we want the user interaction to be temporary or do we want to keep a user profile?

@Shreyanand Yes I think we do want the users to shut down their containers when they exit, but we will keep the PVCs intact. There will be a user profile.
